docs/docs/jobs.md

```diff
@@ -125,6 +125,21 @@ for job in jobs:
     print('Job %s: %s' % (job.id, job.func_name))
 ```
 
+#### Job Status
+
+The status of a job can be one of the following:
+
+* `queued`: The default status for created jobs, except for those that have dependencies, which will be created as `deferred`. These jobs are ready to be executed.
```

Comment:
I think this is better described as `jobs that have been placed in a queue and ready to be executed.`